content/azs.md

```diff
@@ -1,37 +1,58 @@
-!!! note
-    This feature is available with bosh-release v241+. Once you opt into using cloud config all deployments must be converted to use new format. There is no way to opt out of the cloud config once you opt in.
-
-Previously to spread resources over multiple AZs, deployment jobs, resource pools, and networks had to be duplicated and named differently in the deployment manifest. By convention all of these resources were suffixed with `_z1` or `zX` to indicate which AZ they belonged to.
-
-With first class AZs support in the Director it's no longer necessary to duplicate and rename resources. This allows the Director to eliminate and/or simplify manual configuration for balancing VMs across AZs and IP address management.
+!!! Note
+    This feature is available with Bosh v241+ (released on
+    [2015-12-23](https://github.com/cloudfoundry/bosh/releases/tag/stable-3163)).
+    Once you opt into using the “_Bosh v2_” Cloud Config, all deployments must
+    be converted to use new format. There is no way back to “_Bosh v1_”
+    deployment manifests after you've opted in to Cloud Config.
+
+Previously with “_Bosh v1_” deployment manifests, to spread resources over
+multiple availability zones (AZs), deployment jobs, resource pools, and
+networks had to be duplicated and named differently in the deployment
+manifest. By convention, all of these resources were suffixed with `_z1` or
+`zX` to indicate which AZ they belonged to.
+
+With first-class AZs support in the Director, “_Bosh v2_” deployment manifests
+no longer need to duplicate and rename resources. This allows the Director to
+eliminate and/or simplify manual configuration for balancing instances (VMs)
+across AZs and IP address management.
```

Comment:
Given that version 241 was from 2015-2016, I'm wondering if we still need this warning to be so prominent on the page? I do think we should keep this information, but perhaps it can appear towards the bottom of the page in its own section (e.g. "Migrating from / Differences with v1"). What do you think?

Reply:
+1 for having a "Migrating from / Differences with v1" and not placing it so prominently.